data/templates/Detector_StaticSites.csx

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		YOUR_TABLE_NAME
+		| where TIMESTAMP >= startTime and TIMESTAMP <= endTime
+		YOUR_QUERY
+	";
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.Web", resourceTypeName: "staticSites")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), "KUSTO_CLUSTER_NAME", "KUSTO_DB_NAME", null, "GetQuery"),
```

Comment:
static apps is Antares, shouldn't they use ExecuteQuery syntax?

Reply:
Good catch, updated 😊

Reply:
But one thing to note here is - this is onboarded as an ARM Resource. So the Operation Context will not have the stamp in it. Hence the detector authors will have to figure out the stamp of the site (using observer) before invoking ExecuteQuery(..) method.